src/inverseRobot.js

```diff
@@ -25,6 +25,32 @@
  */
 function inverseRobot(robot) {
   // write code here
+  const reverseRobot = {};
+  const robotValues = Object.values(robot);
+  let count = 0;
+  let wordRepeat = 0;
+
+  while (count < robotValues.length) {
+    const value = robotValues[count];
+
+    for (let i = 0; i < robotValues.length; i++) {
+      if (value === robotValues[i]) {
+        wordRepeat++;
+      }
+    }
+    count++;
```

Comment:
let's simplify this, add new values to reverseRobot directly and check for duplicates in ONE for loop